packages/components/src/components/dropdown/_dropdown.scss

```diff
@@ -66,6 +66,36 @@
     }
   }
 
+  .#{$prefix}--dropdown--xl {
+    height: rem(48px);
+    max-height: rem(48px);
+
+    .#{$prefix}--dropdown-text {
+      padding-top: rem(11px);
```

Comment:
Is there any way we can center what is rendered instead of specifying exact pixel values and position relative?